### DISCUSSION TITLE
Why does Parcel (both 1 and 2) keep installing @babel/core in my devDependencies?

### DISCUSSION THREAD

**Discussion by ebrensi** [parcel-bundler/parcel#5097] [General] [answered]
The closest issue I could find was https://github.com/parcel-bundler/parcel/issues/2968.

If I completely wipe the `node_modules` directory fro my project and `npm install`, then watch or serve with parcel, it does not instal `@babel/core`.  But if I do an update or install a new package, then parcel watch or serve, it installs this stupid `@babel/core` which gets on my nerves.  If `@babel/core` is a dependency for Parcel then why can't parcel install it in its own damn `devDependencies` and leave mine alone.  Is this a bug or what? 

edit:  Actually what happens is that when I do a build `parcel serve` or `watch` I get a message in the terminal that it is installing `@babel/core`, and the dev-dependency is added to my project's `package.json` but then it gives me this error:
```
ℹ️ Server running at http://localhost:1234
🚨 Build failed.
@parcel/package-manager: Could not find module "@babel/core", but it was listed in package.json. Run your package manager first.
[mypath]/package.json:42:5
  41 |   "devDependencies": {
> 42 |     "@babel/core": "^7.11.6",
>    |     ^^^^^^^^^^^^^ Defined here, but not installed
  43 |     "documentation": "^13.0.2",
  44 |     "eslint": "^7.8.1",
```
 so it looks like it is not installed.  As far as I can tell, the only way to prevent this is to wipe out my `node_modules` and install from scratch.  It kinda sucks to have to re-install everything every time I update a package.

Can someone tell me why is this happening?

**Comment by devongovett** [comment, marked as answer]
Parcel processes your code with Babel automatically, but in order to give you control over the version of Babel we use, we install it into your project for you rather than hard coding the version into Parcel itself.

**Reply by ebrensi** [reply]
ok got it.  I'll just go ahead and install it as a dev dependency then.
